src/Patches.php

```diff
@@ -297,6 +297,26 @@ public function postInstall(PackageEvent $event)
         $manager = $event->getComposer()->getInstallationManager();
         $install_path = $manager->getInstaller($package->getType())->getInstallPath($package);
 
+        if ($package_name == 'drupal/core' && $install_path == 'docroot/core') {
+            $install_path = 'docroot';
+            if ($this->io->isVerbose()) {
+                $this->io->write(sprintf(
+                    "'%s [id: %s]' installation path has been altered for patching.",
+                    $package->getName(),
+                    $package->getId()
+                ));
+            }
+        } elseif ($package_name == 'drupal/core' && $install_path == 'html/core') {
+            $install_path = 'html';
+            if ($this->io->isVerbose()) {
+                $this->io->write(sprintf(
+                    "'%s [id: %s]' installation path has been altered for patching.",
+                    $package->getName(),
+                    $package->getId()
+                ));
+            }
+        }
+
```

Comment:
Could this be a generalized string munge? So if something used `www/core` or something in the future this path altering would just work?